libs/src/api/user.js

```diff
@@ -137,21 +154,21 @@ class Users extends Base {
    */
   async addUser (metadata) {
     this.IS_OBJECT(metadata)
-    this._validateUserMetadata(metadata)
+    const newMetadata = this._cleanUserMetadata(metadata)
+    this._validateUserMetadata(newMetadata)
 
-    metadata.wallet = this.web3Manager.getWalletAddress()
-    metadata.isCreator = false
```

Comment:
why remove this?

Reply:
`isCreator` isn't even the name of our field in metadata. It should be `is_creator`

Reply:
jk u gud